cookbooks/aws-parallelcluster-common/test/common/libraries/os_properties.rb

```diff
@@ -0,0 +1,55 @@
+#
+# Check if we are running in a virtualized environment
```

Comment:
question, non blocking: is this file repeated for all the cookbooks?

Reply:
Yes, for now. Going to remove it in a future PR.